### PR TITLE
chore(release): small tweaks

### PR DESCRIPTION
**Pull request by Haroenv**
- add contributing guide (based on IS.js one)
- fix small error in api-extractor.json
- better link in CI comment in release script
- replace default with defaultValue: https://api-extractor.com/pages/tsdoc/tag_defaultvalue/
- put todo in separate comments so they aren't found by api-extractor
- avoid using > & {} in tsdoc comments, they can be confused with html apparently, and this looks nicer than the backslashes

### CODE REVIEW

File: packages/client-search/src/types/HasPendingMappingsOptions.ts [hunk 1]
```diff
@@ -2,14 +2,14 @@ export type HasPendingMappingsOptions = {
   /**
    * If the clusters pending mapping state should be on the response.
    *
-   * @default false
+   * @defaultValue false
```
Comment (nunomaduro) [inline packages/client-search/src/types/HasPendingMappingsOptions.ts:5]: What is wrong with `@default`? I thought that was a valid ts-doc annotation.

**Comment by nunomaduro** [comment]
Also seems that lint tests are failing.

File: packages/client-search/src/types/HasPendingMappingsOptions.ts (hunk 1 above)
Comment (Haroenv) [reply packages/client-search/src/types/HasPendingMappingsOptions.ts:5]: as far as I saw, @default is valid only in JSDoc, not in TSDoc + API Extractor as far as I could see: https://github.com/microsoft/tsdoc/issues/27

Comment (nunomaduro) [reply packages/client-search/src/types/HasPendingMappingsOptions.ts:5]: https://api-extractor.com/pages/tsdoc/tag_defaultvalue/

Comment (nunomaduro) [reply packages/client-search/src/types/HasPendingMappingsOptions.ts:5]: Thanks! ( I am sure you did ) make sure you update all `@default` to that new annotations.

Comment (Haroenv) [reply packages/client-search/src/types/HasPendingMappingsOptions.ts:5]: yes, those are the only two

**Pull request merged**